website-frontend/src/routes/research/overview/+page.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+	/** @type {import('./$types').PageData} */
+
+	import { PUBLIC_APIURL } from '$env/static/public';
```

Comment:
I'll keep track on the usage of `PUBLIC_APIURL` in this particular file. Right now, it's a working solution, but we may be performing some crucial changes in the frontend code to minimize the exposure of this variable to `(+page/+layout).svelte` files, and instead dedicate them to either `(+page/+layout).server.ts` or `(+page/+layout).ts` files.